frontend/public/components/search.tsx

```diff
@@ -269,7 +269,8 @@ const SearchPage_: React.FC<SearchProps & StateProps & DispatchProps> = (props)
                 >
                   {getToggleText(resource)}
                   {props.perspective !== 'admin' && (
-                    <a
+                    <button
```

Comment:
Can we use the Patternfly button here? Then we wouldn't have to use the `pf-c-button`, `pf-m-link` classes here.

Reply:
Good idea, changed to PatternFly Button